test-requirements.txt

```diff
@@ -6,6 +6,7 @@ pytest-localserver==0.5.0
 pytest-cov==2.8.1
 jsonschema==3.2.0
 pyrsistent==0.16.0 # TODO(py3): 0.17.0 requires python3, see https://github.com/tobgu/pyrsistent/issues/205
+mock # for testing under python < 3.3
```

Comment:
Instead of doing this please just use the pytest monkeypatch fixture

Reply:
I'm not sure how you mean. I do see how you could use `monkeypatch` for `mock.patch.object` calls, but how would you handle getting `mock.Mock` instances when you need them?

Reply:
I don't know how you're going to use mock but I've just been creating my own stub classes. I find a lot of mock's behavior too magical wrt attribute access on the mock obj